boggle/Boggle_works.py

```diff
@@ -0,0 +1,152 @@
+
+from random import *
+from math import *
+from Gui import *
+import string
+
+a = range(1,17)

```

Comment:
"a" is a bad variable name, as I have no idea what it is. i,j,k are accepted for-loop indicies, and x,y,z are generally ok for coordinates, but always better to use longer names.

Especially for a global variable like this.